docs/pages/build/framework/micro-rollup.mdx

```diff
@@ -38,17 +38,6 @@ Currently the Micro-Rollup supports only 1 state machine. This will be extended
 
 The Micro-Rollup object also takes in a few optional parameters
 
-1. `isSandbox` - A boolean flag to indicate if the rollup is running in a sandbox environment,
+1. `stfSchemaMap` - A map of the schema names to the schema objects. This is used to map the schema names to the state transitions of the rollup.
```

Comment:
Add a note that, defining this helps in playground interactions as schema are auto-picked for reducer